src/main/java/com/liefery/android/stop_badge/StopBadge.java

```diff
@@ -233,12 +233,15 @@ int shadowSizeX() {
     }
 
     int shadowSizeY() {
-        return (int) ( shadowRadius + Math.abs( shadowDx ) );
+        return (int) ( shadowRadius + Math.abs( shadowDy ) );
     }
 
     public Bitmap export( int size ) {
-        Bitmap bitmap = Bitmap.createBitmap( size + shadowSizeX() * 2, size
-            + shadowSizeY() * 2, Bitmap.Config.ARGB_8888 );
+        int newSize = Math.max( shadowSizeX(), shadowSizeY() ) * 2 + size;
```

Comment:
I don't see what we're improving here. We're are giving the Bitmap way more space than necessary, but we should be as humble as possible when working with memory intensive Bitmaps. Or is that fixing an actual issue?